src/Makefile

```diff
@@ -96,7 +96,7 @@ $(LIBTRIEMAP):
 .PHONY: $(LIBTRIEMAP)
 
```

Comment:
We should return to using the rm_malloc family, which I think would require the additional `-DCMAKE_C_FLAGS`:
`-DREDIS_MODULE_TARGET -DMALLOC=rm_malloc -DCALLOC=rm_calloc -DREALLOC=rm_realloc -DFREE=rm_free`

An additional include of `rmalloc.h` is also necessary, which we used to do in what would now be `deps/GraphBLAS/Source/GB.h:393`, but might also be viable as a `-I` CMAKE_C_FLAG.